### ISSUE TITLE
Test-VersionPattern function failing on ThinkCenter M625q

### ISSUE THREAD

**Issue by MadhuSunke417** [jantari/LSUClient#42]
### Version

1.3.3

### Computer Model

ThinkCenter M625q

### Problem

Test-VersionPattern function failing on ThinkCenter M625q. please find the attached image for more info.

### Additional context

<img width="576" alt="ThinkCenter" src="https://user-images.githubusercontent.com/14003673/143306451-3b3ab3b0-0ec5-4d5c-91f1-ef99230bc1e2.png">

**Comment by jantari** [comment]
Thanks for the bug report, the error occurs because in [this package](https://download.lenovo.com/pccbbs/thinkcentre_drivers/u6vdodcus17_tc_m625q_2_.xml) the `_WindowsBuildVersion` has two values, which I don't handle properly because I hadn't seen it before:

```xml
<_WindowsBuildVersion>
  <Version>18362</Version>
  <Version>18363</Version>
</_WindowsBuildVersion>
```

Should be an easy fix though.

**Comment by MadhuSunke417** [comment]
do I need to modify anything on module or can we expect new version release ?

**Comment by jantari** [comment]
New release for sure, but can't say exactly when. I just pushed the fix for this to the [develop branch](https://github.com/jantari/LSUClient/tree/develop) though, so if you want to get it early you can download it now from there.

**Comment by jantari** [comment]
Fixed in Version 1.3.4 🎉
